README.md

```diff
@@ -190,6 +190,9 @@ Use Android Cache Fix Plugin 2.4.6 when using an older Android Gradle Plugin ver
 
 * Supported Gradle versions: 5.4.1+
 * Supported Android Gradle Plugin versions: 3.5.4, 3.6.4, 4.0.1, 4.1.3, 4.2.2
+
+### Older Kotlin Gradle Plugin Versions
+Use Android Cache Fix Plugin 2.6.0 when using an older Kotlin Gradle Plugin version.
```

Comment:
Let's be specific here with the version and list the range on the line below.

Reply:
done